be/nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ignore": [".git", "node_modules", "dist", "build"],
+  "watch": ["./src"],
+  "exec": "npm run build:start",
```

Comment:
Should we have the `generate` here too?

Reply:
@ja-fra I would say no, because if you generate the files newly you want the server to restart.